daemon/algod/api/server/v2/handlers.go

```diff
@@ -270,7 +278,9 @@ func (v2 *Handlers) AccountInformation(ctx echo.Context, address string, params
 		}
 	}
 
-	account, err := AccountDataToAccount(address, &record, assetsCreators, lastRound, amountWithoutPendingRewards)
+	minBalance := record.MinBalance(&consensus)
```

Comment:
no need to calculate minBalance here. record of the basics.AccountData is passed to `AccountDataToAccount` so it can call `MinBalance` there ? I'd better pass consensus params there since it also needed for some other calculated fields if we decide to add them later.

Reply:
Will try and incorporate your suggestions. Stay tuned.